cmd/quadlet/main.go

```diff
@@ -322,9 +325,24 @@ func main() {
 		noKmsg = true
 	}
 
+	var prevError error
+	defer func() {
+		if prevError != nil {
+			Logf("%s", prevError)
+			os.Exit(1)
+		}
+		os.Exit(0)
+	}()
```

Comment:
Nit: I would find it cleaner to create a new function that is being called from main.  At the moment, the error handling is a bit hidden in the deferred func here which feels less idiomatic.

Then we can do a:
```go
func main() {
   if err := newFunc(); err != nil {
       Logf("%s", err)
       os.Exit(1)
   }
}
```